### PR TITLE
Added test routine strSearchLog --not completed

### PR DESCRIPTION
**Pull request by JackieJoo**

### CODE REVIEW

File: proto/dwtools/abase/l5.test/StringTools.test.s [hunk 1]
```diff
@@ -4796,6 +4796,894 @@ function strSearchOptionOnTokenize( test )
...
+  ]
+  var expectedLog = '1 : abcabcabc\n1 : abcabcabc\n1 : abcabcabc';
+  var got = _.strSearchLog( input );
+  test.identical( got.parcels, expectedParcels );
+  test.identical( got.log, expectedLog );
+
+  /* */
+
+  test.case = 'src - empty string, ins - regexp for empty strings';
+  var input = { src : '', ins : /(?:)/g, onTokenize : _.strTokenizeJs, excludingTokens : 'name', gray : 1  };
```
Comment (dmvict) [inline proto/dwtools/abase/l5.test/StringTools.test.s:5116]: Please, use single style for input arguments. 
```js
var input =
{
  src : '',
  ins : /(?:)/g,
  onTokenize : _.strTokenizeJs,
  excludingTokens : 'name',
  gray : 1
};
```

File: proto/dwtools/abase/l5.test/StringTools.test.s [hunk 2]
```diff
@@ -4796,6 +4796,894 @@ function strSearchOptionOnTokenize( test )
...
+
+  test.case = 'src - empty string, ins - regexp';
+  var input = { src : '', ins : /x/, onTokenize : _.strTokenizeJs, excludingTokens : 'name', gray : 1 };
+  var got = _.strSearchLog( input );
+  console.log(got)
+  var expectedParcels = [];
+  var expectedLog = '';
+  test.identical( got.parcels, expectedParcels );
+  test.identical( got.log, expectedLog );
+
```
Comment (dmvict) [inline proto/dwtools/abase/l5.test/StringTools.test.s:5139]: Please, remove empty line.

File: proto/dwtools/abase/l5.test/StringTools.test.s [hunk 3]
```diff
@@ -4796,6 +4796,894 @@ function strSearchOptionOnTokenize( test )
...
 
+function strSearchLog( test )
+{
+  test.open( 'one line' );
+
+  test.case = '1 letter match at the start - one line'
+  var input = 
+  {
+    src : 'abcdefg',
+    ins : [ 'a' ],
```
Comment (dmvict) [inline proto/dwtools/abase/l5.test/StringTools.test.s:4807]: Please, add test case with empty line : 
```js
ins : [ '' ]
```
